src/shadydom.js

```diff
@@ -70,15 +68,24 @@ if (utils.settings.inUse) {
     'preferPerformance': utils.settings['preferPerformance'],
     // Integration point with ShadyCSS to disable styling MutationObserver,
     // as ShadyDOM will now handle dynamic scoping.
-    'handlesDynamicScoping': true
+    'handlesDynamicScoping': true,
+    'wrap': utils.settings.noPatch ? wrap : (n) => n,
```

Comment:
Oh I wrote below a solution for the tests, but I now see that this was incorporated here. :+1: